velox/functions/prestosql/DecimalVectorFunctions.cpp

```diff
@@ -22,6 +22,17 @@
 namespace facebook::velox::functions {
 namespace {
 
+struct ExtraParams {
+  ExtraParams() {}
```

Comment:
We don't need constructors for these plain struct and union, just do `ExtraParams extraParams{}` should be enough

Reply:
I want to initialize the optional parameter: `params.round.decimal.reset();`, during testing, the has_value() behavior is flaky if not initialized, at least on mac.